packages/xo-server/src/xo-mixins/backups-ng/index.mjs

```diff
@@ -484,6 +484,21 @@ export default class BackupNg {
         })
       }
       return result.id
+    } catch (error) {
+      if (
+        // xxhash is the new form consistency hashing in CH 8.1 which uses a faster,
+        // more efficient hashing algorithm to generate the consistency checks
+        // in order to support larger files without the consistency checking process taking an incredibly long time
+        error.code === 'IMPORT_ERROR' &&
+        error.params?.some(
+          param =>
+            param.includes('INTERNAL_ERROR') &&
+            param.includes('Expected to find an inline checksum') &&
+            param.includes('.xxhash')
```

Comment:
Why not just check `Ref:390/00000000.xxhash` ? 
It looks more like an error identifier.

Reply:
`Ref:390` isn't a constant. It could be `Ref:<any number>`.